README.md

```diff
@@ -1 +1,27 @@
-# cruise-ships-LouiseOsborne
+# Cruise Ships 
+
+Cruise Ships is a JavaScript project as part of the Command Shift Software Engineer Bootcamp. 
+It is an exercise in object-oriented programming and test driven development with jest. 
+
+Ships follow an Itinerary and can dock and depart various Ports along the way. 
+
+# Learning
+
+- Object-oriented programming
+- Test driven development, using jest. 
+- Domain modelling
+- Single responsibility principle 
+- Dependency inversion
+- Mocks, stubs and spies
+
+# Installation
+
+Create a fork and clone the repo. 
+Run npm install to download dependencies. 
+
+# Usage
+Coming Soon.
+
+# Authors and acknowledgement
+
+Louise Osborne, as part of the Command Shift Software Engineer Bootcamp curriculum. 
```

Comment:
Decent README. I would also add a dependencies section. In this case just jest. Something like

## Dependencies
### Dev Dependencies
- [Jest](https://jestjs.io/)

You can make a link by wrapping test text in [] and then following that wrap the link in () 
e.g. [Jest] (https://jestjs.io/) but without the space between ] and (. 
Try to link to somewhere that gives more information about the dependencies. 

You can also add code snippets using back ticks `. This will make your commands stand out more, 
e.g. `npm install`

Reply:
I like to use [this online markdown editor](https://jbt.github.io/markdown-editor/) so I can see the formatting in real time

Reply:
This is brilliant, thank you!